src/cli/query.rs

```diff
@@ -61,6 +62,12 @@ impl Command for Query {
                 "the scope to query against",
                 None,
             )
+            .named(
+                "params",
```

Comment:
query-params perhaps? I don't know, just feel like using params might come back to bite us in the future?

Reply:
In the docs they are called place-holder parameters, but that feels too long winded. Similarly query-params seems a bit lengthy. What is your concern with params?